src/config.ts

```
@@ -16,7 +16,7 @@ export const botConfig = {
 // Proxy configuration
 export const proxyConfig = {
   host: process.env.PROXY_HOST || "0.0.0.0",
-  port: parseInt(process.env.PROXY_PORT || "8765"),
+  port: parseInt(process.env.PROXY_PORT || "4000"),
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)